AppController/djinn.rb

```diff
@@ -4774,6 +4774,9 @@ def prep_app_dashboard()
       setup_app_dir(AppDashboard::APP_NAME, true)
     }
 
+    # Allow fewer dashboard instances for small deployments.
+    min_dashboards = [3, get_all_appengine_nodes.length].min
```

Comment:
If we have 100 appengine machines, I don't think we want 100 dashboards. Now that we no longer send logs there, I can't think of a use case that would require more than 1 to handle the load. The only reason we start 3 is for fault tolerance, which probably isn't a priority for someone running a single-node deployment.